django_camunda/dmn/datastructures.py

```diff
@@ -10,6 +12,6 @@ class DMNVariable:
 
 
 @dataclass
-class IntrospectionResult:
-    inputs: List[DMNVariable]
-    output: Dict[str, DMNVariable]
+class DMNIntrospectionResult:
+    inputs: List[DMNInputParameter]
+    outputs: List[DMNOutputParameter]
```

Comment:
This can be deleted completely I think.

Reply:
I left it because I wanted to use it in OF :thinking: Maybe I should move it there?

Reply:
Or maybe I should add a method to the parser `extract_parameters` that returns this :thinking:

Reply:
the latter is nicest IMO :)